scripts/nesi_scripts/adv_im/run_adv_im_obs_maui.sl

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+# script version: slurm
+# im_calc
+#
+
+# Please modify this file as needed, this is just a sample
+#SBATCH --account=nesi00213
+#SBATCH --ntasks=40
+#SBATCH --time=24:00:00
+#SBATCH --output im_calc_%j_%x.out
+#SBATCH --error im_calc_%j_%x.err
+#SBATCH --nodes=1
+#SBATCH --hint=nomultithread
+#SBATCH --exclusive
+
+export IMPATH=$gmsim/IM_calculation/IM_calculation/scripts
+export PYTHONPATH=$gmsim/qcore:/$PYTHONPATH:$IMPATH
+
+if [[ $# -lt 2 ]];
+then
+    echo "please provide 1. path to obs_dir 2. list of event names"
+    exit 1
+fi
+
+obs_dir=$1
+list_event=$2
+opensees_bin=${3:-/nesi/project/nesi00213/opt/maui/tmp/OpenSees}
```

Comment:
Default value?

Reply:
yeah, the :- basically means if none then use this.